src/Deals.php

```diff
@@ -101,13 +101,18 @@ public function getBySearch($search, $limit = false, $offset = 0)
 		return $this->formatDealsForReturn($deals);
 	}
 
-	public function getBaseQuery($limit = false, $offset = 0)
+	public function getBaseQuery($limit = false, $offset = 0, $order = 'rating')
 	{
 		$query = DB::table('deals')
 					->select(DB::raw('fmtc_merchants.*, fmtc_deals.*'))
 					->join('merchants', 'deals.nMerchantID', '=', 'merchants.nMerchantID')
-					->where('dtEndDate', '>=', date('Y-m-d H:i:s'))
-					->orderBy('fRating', 'desc');
+					->where('dtEndDate', '>=', date('Y-m-d H:i:s'));
+		
+		if ($order == 'expiration') {
```

Comment:
Nate, I'm wondering if we should be creating a conditional for ordering. I think it might make more sense and be more flexible to have 2 extra parameters in the getBaseQuery function: $orderField = 'fRating' and $desc = true.

What do you think?